src/components/AudioAnalysis/PitchSelect.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
```

Comment:
@Mattallmighty i think this is an old module (jsx) and there is already a tsx duplicate.
shall we remove this file?